hls4ml/model/hls_model.py

```diff
@@ -467,8 +467,11 @@ def replace_node(self, old_node, new_node):
 
         self.graph = OrderedDict((new_node.name, new_node) if k == old_node.name else (k, v) for k, v in self.graph.items())
 
-    def get_weights_data(self, layer_name, var_name):
-        return self.reader.get_weights_data(layer_name, var_name)
+    def get_weights_data(self, layer_name, var_name, module_name=None):
+        if module_name is not None:
+            return self.reader.get_weights_data(layer_name, var_name, module_name)
+        else:
+            return self.reader.get_weights_data(layer_name, var_name)
```

Comment:
Changes here: https://github.com/abdelabd/hls4ml/commit/ca49221d672c2712b8b86afe9ba7a2a6c43555f0

I made the changes to the PyTorchModelReader.get_weights_data() because this seemed like a light change, and the PygModelReader inherits this method. If you want, I can just implement this as a method unique to PygModelReader instead.